tests/src/test/java/alluxio/master/MasterTestUtils.java

```diff
@@ -60,7 +62,7 @@ private static MasterRegistry createFileSystemMasterFromJournal(boolean isLeader
     MasterRegistry registry = new MasterRegistry();
     JournalFactory factory = new MutableJournal.Factory(new URI(masterJournal));
     new BlockMaster(registry, factory);
-    new FileSystemMaster(registry, factory);
+    new DefaultFileSystemMaster(registry, factory);
```

Comment:
I would prefer to use the `FileSystemMasterFactory` here to limit the number of places where we directly instantiate `DefaultFileSystemMaster`